lib/tasks/scripts/get_bsnl_account_balance.rb

```diff
@@ -0,0 +1,27 @@
+class GetBsnlAccountBalance
+  # (20_000 * 3 patients from current experiments + 15_000 patients from stale experiments) * 4 segments per message
+  # We're unlikely to actually hit this number on any given day, but we'd like to alert early if we're close to the limit
+  MAX_DAILY_MESSAGE_SEGMENT_COUNT = 75_000 * 4
+  BALANCE_EXPIRY_ALERT_DAYS = 3
```

Comment:
@prabhanshuguptagit can you verify this math looks good?

Also setting `BALANCE_EXPIRY_ALERT_DAYS` to be low deliberately, can bump it up to a week once the larger recharge goes through